waku/v2/protocol/waku_message.nim

```diff
@@ -27,8 +27,8 @@ type
   ContentTopic* = string
 
 const 
-  DefaultPubsubTopic*: PubsubTopic = "/waku/2/default-waku/proto"
-  DefaultContentTopic*: ContentTopic = "/waku/2/default-content/proto"
+  DefaultPubsubTopic*: PubsubTopic = PubsubTopic("/waku/2/default-waku/proto")
+  DefaultContentTopic*: ContentTopic = ContentTopic("/waku/2/default-content/proto")
```

Comment:
This is redundant and not necessary, the types are not `distinct string`

Reply:
yep, thought it was nice for reference, so that people declaring topics they would see this and follow the pattern:

```nim
let x = PubsubTopic("x/y/z/a")
```

instead of `let y = "x/y/z/a"` which I would say its bad practise?

no strong opinion about it, though :)

Reply:
It is redundant since the variable is already type annotated:

```nim
 DefaultPubsubTopic*: PubsubTopic = PubsubTopic("/waku/2/default-waku/proto")
 #~~~~~~~~~~~~~~~~~~~~~~~~~^
````